source/concepts.rst

```diff
@@ -34,6 +34,8 @@ Semantic types also help users avoid semantically incorrect analyses. For exampl
 
 Check out the :doc:`semantic types page <semantic-types>` for more information about semantic types and what types are currently available.
 
+.. _`plugins`:
```

Comment:
I don't see a :ref: for this below... is this used?

Reply:
nope! i wrote this before I knew how to link to the list of plugins...